### PR TITLE
Conduit Changes

### PR DESCRIPTION
**Pull request by anandhis**
Requesting pull for conduit changes made.  Both oob and ofi-ethernet tested and working.

### CODE REVIEW

**Comment by rhc54** [comment]
soooooo...this is the one and only PR to be considered? I take it that #14 is defunct?

**Comment by anandhis** [comment]
yes

File: orte/mca/rml/base/base.h [hunk 1]
```diff
@@ -224,6 +226,22 @@ OBJ_CLASS_DECLARATION(orte_rml_recv_request_t);
         opal_event_active(&(m)->ev, OPAL_EV_WRITE, 1);          \
     } while(0);
 
+/*
+ reactivates rcv msg on the unposted rcvd list when a match occurs
+ need a different path as the QoS recv processing was already done
+ for this process
+*/
+#define ORTE_RML_REACTIVATE_MESSAGE(m)                         \
```
Comment (rhc54) [inline orte/mca/rml/base/base.h]: we don't need this macro any more as the qos is gone

Comment (rhc54) [inline orte/mca/rml/base/base.h]: can remove it later - no harm either way

**Pull request merged**